packages/cli/src/tokens/build/builtin-colors.css

```diff
@@ -0,0 +1,92 @@
+@layer ds.theme.color {
```

Comment:
We should also build this from tokens, as It _can_ potentially change, or if we do some other changes in regards to `@layer` or data-attributes.

Reply:
Is it okay to postpone this, and use hardcoded `builtin-colors` for now? The easiest way to do it would be adding a `builtin-color` mode in design-tokens, in addition to `main-color` and `support-color`. But I need to discuss with @Febakke how this should work in Figma.

Reply:
@mimarz [Here's an example](https://github.com/digdir/designsystemet/commit/84a738bae96ed4f43f42d9c926b7a8e6e265e88c) on how it could be done without adding more modes in the token structure, but I don't like how complicated the filter function in `formats/css.ts` becomes 😵